avocado/plugins/jsonresult.py

```diff
@@ -62,6 +62,7 @@ def _render(self, result):
                    'errors': result.errors,
                    'failures': result.failed,
                    'skip': result.skipped,
+                   'cancel': result.cancelled,
```

Comment:
OK, but it would also be nice to add checks for this field on JSON related functional tests.